lmdeploy/messages.py

```diff
@@ -113,6 +113,7 @@ class TurbomindEngineConfig:
             For versions of lmdeploy between `v0.2.0` and `v0.2.1`, it defaults to 0.5, depicting the percentage of TOTAL GPU memory to be allocated to the k/v cache.
             For lmdeploy versions greater than `v0.2.1`, it defaults to 0.8, signifying the percentage of FREE GPU memory to be reserved for the k/v cache
         cache_block_seq_len (int): the length of the token sequence in a k/v block, default to 64
+        enable_prefix_caching (bool): enable cache and reuse blocks with prefix matching, default to False
```

Comment:
The current implementation is the cache of system prompts. Because the significance of caching decoding results is not high and the probability of being matched is low, it is best to clarify that it is `caching system prompts` here.

Reply:
Currently, the scope of cache is different from that in https://github.com/InternLM/lmdeploy/pull/1429. Whether the two should have consistent behavior needs to be discussed @grimoire @lzhangzz @lvhan028

Reply:
Now we only cache the prompt blocks (for stateful inference, it's the first turn prompt) to keep the block trie concise and reduce the eviction times. Because the matching rate for generated text and long dialogue in stateful inference is much lower.

Reply:
I think the difference is acceptable. I can add another flag to disable caching generated token.